exercises/1-1-getting-started/1-Install-python.md

```diff
@@ -0,0 +1,54 @@
+# 1. Installing a Python interpreter
+
+It is useful to learn different strategies to install the Python interpreter in a system, even if there is already a working Python installation. Here are some options, from the most system-specific to the most Python-specific way:
+
+1. Use your system native package system (e.g. `apt`, `pacman`, ...)
+2. Use an alternative package manager available in your system (e.g. `brew`, `conda`, `spack`, ,...)
+3. Use a development tool managing different programing tools versions (e.g. `asdf`, ...)
+4. Use a Python version manager (e.g. `uv`, `pyenv`) to kickstart a Python installation without ex
```

Comment:
Wait, Python had an ex?

Reply:
Hey, we all have a  past, don't we? :wink:  Fixed.